napari/layers/tracks/_track_utils.py

```diff
@@ -66,31 +71,19 @@ class TrackManager:
         Track ID for each vertex in track_vertices.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, data: np.ndarray) -> None:
         # store the raw data here
-        self._data = None
+        self.data = data
```

Comment:
That makes `data` a public attribute, so I'd like the +1 of some core devs.

Reply:
Yep, through the setter :)